src/cascadia/TerminalSettingsEditor/Utils.cpp

```diff
@@ -28,7 +28,7 @@ namespace winrt::Microsoft::Terminal::Settings
     hstring LocalizedNameForEnumName(const std::wstring_view sectionAndEnumType, const std::wstring_view enumValue, const std::wstring_view propertyType)
     {
         // Uppercase the first letter to conform to our current Resource keys
-        auto fmtKey = fmt::format(L"{}{}{}/{}", sectionAndEnumType, char(std::towupper(enumValue[0])), enumValue.substr(1), propertyType);
+        auto fmtKey = fmt::format(FMT_COMPILE(L"{}{}/{}"), sectionAndEnumType, enumValue, propertyType);

```

Comment:
If you got here, you're 3/4ths done with everything but `ActionArgs.cpp`. I'd suggest skipping `ActionArgs.cpp` first, because it's unfortunately quite something.

Reply:
This change seems dangerous; how did you settle on making it?

"Chesterton's fence"

Reply:
FWIW: resource keys seem to be case sensitive. That's why we put the enum in title case with the weird "toupper" thing.

I don't know why somebody used the dispreferred form of casting, `char(x)`

Reply:
`char(std::towupper(enumValue[0]))` doesn't work anymore, because it now prints an integer (because the builtin char -> wchar support was removed).

I've considered just removing the `char` cast, but I was curious whether it would work without this cast. I went through every view in the settings UI and none of them threw an exception.

Reply:
But the enum names come from our IDL (through jsoncpp), so why would anything change?

Reply:
Did you confirm that it works on Windows 10?